src/base/bittorrent/private/resumedatasavingmanager.cpp

```diff
@@ -54,3 +54,19 @@ void ResumeDataSavingManager::saveResumeData(QString infoHash, QByteArray data)
         }
     }
 }
+
+void ResumeDataSavingManager::saveTorrentsQueue(const QByteArray &data) const
+{
+    const QString filename = QLatin1String {"queue"};
+    const QString filepath = m_resumeDataDir.absoluteFilePath(filename);
+
+    qDebug() << "Saving torrents queue in" << filepath;
```

Comment:
just asking, do you really intend to have this line?

Reply:
No.